From 3659eef7e54958b1608231cc4d35714a8b5452a5 Mon Sep 17 00:00:00 2001
From: "Dr. Greg" <greg@enjellic.com>
Date: Sun, 28 Mar 2021 09:11:59 -0500
Subject: [PATCH 39/50] Drop pseudonym bit identifier.

With the updated pseudonym architecture that registers an
integrity inode pointer as an indication of a pseudonym in a
namespace, there is no need for the bit entry in the flags
variable to indicate an integrity inode is a pseudonym.  Drop the
declaration for the bit member.
---
 security/integrity/integrity.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/security/integrity/integrity.h b/security/integrity/integrity.h
index 0e8ab7f2b34d..16ef95ba8cce 100644
--- a/security/integrity/integrity.h
+++ b/security/integrity/integrity.h
@@ -33,7 +33,6 @@
 #define EVM_IMMUTABLE_DIGSIG	0x08000000
 #define IMA_FAIL_UNVERIFIABLE_SIGS	0x10000000
 #define IMA_MODSIG_ALLOWED	0x20000000
-#define IMA_PSEUDONYM		0x40000000
 
 #define IMA_DO_MASK		(IMA_MEASURE | IMA_APPRAISE | IMA_AUDIT | \
 				 IMA_HASH | IMA_APPRAISE_SUBMASK | IMA_MAP)
-- 
2.31.1

